tests/shared/utils.spec.js

```diff
@@ -333,4 +333,32 @@ describe('Shared: Utils', function() {
 
   });
 
+  describe('.isValidPercentage()', function() {
```

Comment:
Could also test it with floating-point numbers? E.g. -0.1, 0.1, 12.3, 99.9, 100.1